### ISSUE TITLE
Feedly / New Feed: detect situation when someone has not created any folders yet

### ISSUE THREAD

**Issue by brentsimmons** [Ranchero-Software/NetNewsWire#1448]
With Feedly, all feeds need to be inside a folder. If that person has no folders, and they try to add a feed, they won’t be able to.

We should let them know what the problem is before they try adding the feed, and offer to create a folder. (This would be in the Add Feed screen.)

**Comment by vincode-io** [comment]
Related feedback from TestFlight:

> I’m unable to add a feed to my Feedly account, because the app tries to add it directly to “uncategorised”. I don’t organise my feeds, so there’s no way for me to add a new one as the app seems to require a folder.

**Comment by vincode-io** [comment]
@kielgillard Is there some way that a user can add feeds to Uncategorized using the Feedly API?  It seems that some users don't organize their feeds and just expect them to be in a lump in side the Uncategorized smart feed (see my above comment).

**Comment by kielgillard** [comment]
Huh. I just checked the Feedly web app and it seems you most certainly can add feeds to the uncategorized collection. I also checked the API documentation and I cannot seem to find where it says you cannot add feeds to this collection. If I remember correctly, this impression that you cannot change the uncategorized collection directly came from Feedly's API -- it'd return a 400 status code or something like that.

I'll look into this for NNW as soon as possible.
